commit b4d5a8f223726f2f4ae3fcc8c2296ac1ad943694
Author: Ilya Tumaykin <itumaykin@gmail.com>
Date:   Fri Jul 7 09:01:04 2017 +0300

build: fix icons install check

Otherwise exists(./images/application/cmst-icon.png) check is ignored.

diff --git a/cmst.pro b/cmst.pro
index 609bc7c..1aa019e 100755
--- a/cmst.pro
+++ b/cmst.pro
@@ -26,7 +26,7 @@ documentation.extra = gzip --force --keep ./misc/manpage/cmst.1
 INSTALLS += documentation

 # application icons - 
-exists(./images/application/cmst-icon.png) 
+exists(./images/application/cmst-icon.png) {
 	LIST = 16 20 22 24 32 36 40 48 64 72 96 128 192 256 384 512
 	for(a, LIST) {
 		icon$${a}.path = /usr/share/icons/hicolor/$${a}x$${a}/apps
@@ -38,7 +38,7 @@ exists(./images/application/cmst-icon.png)
 		iconsvg.files = ./images/application/scalable/cmst.svg
 		INSTALLS += iconsvg
 	}
-else {
+} else {
 	system(sed -i 's/Icon=cmst/Icon=preferences-system-network/g' "./misc/desktop/cmst.desktop")
 	system(sed -i 's/Icon=cmst/Icon=preferences-system-network/g' "./misc/desktop/cmst-autostart.desktop")
 }
